app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  before_save :email_downcase
+
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  validates :name, presence: true, length: { maximum: Settings.name_maximum }
```

Comment:
bên trong {} ko để thừa space nhé
VD:
```suggestion
  validates :name, presence: true, length: {maximum: Settings.name_maximum}
```